zjb/server.py

```diff
@@ -18,9 +18,10 @@
 template_index = j2env.get_template('index.html.j2')
 template_results = j2env.get_template('results.html.j2')
 
+session = None
 
 def get_results(filters: dict) -> dict:
-    session = db.session()
+    global session = db.session()
```

Comment:
What's with those globals...? Why?

Reply:
indeed no need.  initially thought to fetch from the database during the call to get_results. when realized how python code is invoked from the javascript, the  fetch_from_database was created as @app.route('/fetch-from-database', methods=['GET']). And, BTW, it has to open its own session. Anyways, turning to this change to draft up until making sure it's working.